src/components/victory-group/victory-group.js

```diff
@@ -404,6 +411,7 @@ export default class VictoryGroup extends React.Component {
         theme: child.props.theme || props.theme,
         labelComponent: props.labelComponent || child.props.labelComponent,
         style,
+        domainPadding: {x: (props.width / noOfChildren) / 2},
```

Comment:
@boygirl so expected behavior would that it would look the same (with groups hanging over the end) unless an xOffset is provided?